composer.json

```diff
@@ -137,9 +137,6 @@
         "classmap-authoritative": true,
         "preferred-install": "dist",
         "process-timeout": 20,
-        "platform": {
-            "php": "7.4.13"
```

Comment:
Wasn't this here for renovate?

Reply:
Yeah, I don't think it did anything though. PHP 8.1 is next on the agenda regardless.